packages/rich-text/src/to-dom.js

```diff
@@ -283,6 +292,11 @@ export function applySelection( selection, current ) {
 		range.setEnd( endContainer, endOffset );
 	}
 
+	// If ranges the live range is the same, there's no need to remove and add.
+	if ( isRangeEqual( range, windowSelection.getRangeAt( 0 ) ) ) {
```

Comment:
Are we guaranteed to have `selection.rangeCount > 0` ? It's been a source of thrown errors in the past.

Reply:
Yeah, seems fair to have this condition.

Reply:
`windowSelection.getRangeAt( 0 )` - I saw this throwing an error earlier today

Reply:
![screen shot 2018-12-03 at 15 02 39](https://user-images.githubusercontent.com/699132/49378294-82bea300-f70c-11e8-99c9-aaa9ef5e9d10.png)

it was a different method. It's hard to reproduce:
1. Select all blocks.
1. Cut all blocks.
1. Paste it.
1. Use More Menu and copy content menu item.

Reply:
Could you create a separate issue?

Reply:
Yes, sure 👍